src/sync/index.js

```diff
@@ -181,6 +181,16 @@ async function fetchNameOptionsFromTopic(db, address) {
   }
 }
 
+async function fetchTopicAddressFromOracle(db, address) {
```

Comment:
isnt it `fetchOracleByAddress`?

Reply:
I just kinda went with the same naming as the other `fetchNameOptionsFromTopic` method.